### DISCUSSION TITLE
Table definition has changed, please retry transaction when dumping table `model_has_permissions` at row: 0

### DISCUSSION THREAD

**Discussion by devr-team** [spatie/laravel-backup#1465] [Q&A]
Hello everyone,

I'm getting this error when I'm running backup:run:

Illuminate\Foundation\Console\Kernel->handle()
#24 {main}
Backup failed because: The dump process failed with exitcode 3 : Unknown error : mysqldump: Error 1412: Table definition has changed, please retry transaction when dumping table `model_has_permissions` at row: 0

Backup is running on a slave database.

Anyone any idea?

Regards

**Comment by PaolaRuby** [comment]
[Error 1412: Table definition has changed](https://www.google.com/search?q=Error+1412%3A+Table+definition+has+changed&oq=Error+1412%3A+Table+definition+has+changed)
